test/builder/auto_mocks_test.dart

```diff
@@ -3037,79 +3037,55 @@ void main() {
   });
 
   test(
-      'throws when GenerateMocks is given a class with a getter with a '
-      'non-nullable class-declared type variable type', () async {
-    _expectBuilderThrows(
-      assets: {
-        ...annotationsAsset,
-        ...simpleTestAsset,
-        'foo|lib/foo.dart': dedent('''
+      'GenerateMocks works given class with a getter with a non-nullable '
```

Comment:
Instead of 'works', we can say something more specific, like

`'uses defaultDummyValue, given a class...'`

Same with the 'works' text in the tests below.